src/bin/vip-wp.js

```diff
@@ -172,6 +172,12 @@ const launchCommandAndGetStreams = async ( { guid, inputToken, offset = 0 } ) =>
 		console.log( 'There was an error with the authentication:', err.message );
 	} );
 
+	socket.on( 'cancel', message => {
+		console.log( `Cancel received from server: ${ message }` );
+		socket.close();
+		process.exit();
```

Comment:
Should we use a non-zero exit code?

Reply:
Addressed in beb644ed7345a56831e1b7db3f4b6a238706e390